packages/now-cli/scripts/build.ts

```diff
@@ -1,61 +1,10 @@
 import cpy from 'cpy';
-import tar from 'tar-fs';
 import execa from 'execa';
 import { join } from 'path';
-import pipe from 'promisepipe';
-import { createGzip } from 'zlib';
-import {
-  createWriteStream,
-  mkdirp,
-  remove,
-  writeJSON,
-  writeFile,
-} from 'fs-extra';
-
-import { getDistTag } from '../src/util/get-dist-tag';
-import pkg from '../package.json';
-import { getBundledBuilders } from '../src/util/dev/get-bundled-builders';
+import { remove, writeFile } from 'fs-extra';
```

Comment:
Can we also replace `fs-extra` in favor of `fs.promises`?

https://nodejs.org/dist/latest-v10.x/docs/api/fs.html#fs_fs_promises_api

Reply:
Sure, but it's used in Now CLI itself in quite a few places as well. Maybe we can tackle all at once in a separate PR?

Reply:
True dat 👍